server/models/Request.js

```diff
@@ -0,0 +1,35 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema({
+    user_id:{
+        type: String,
+        required: true,
+      },
+    sitter_id: {
+        type: String,
+        required: true,
+    },
+    start: {
+        type: Date,
+        required: true
+    },
+    end:{
+        type: Date,
+        required: true
+    },
```

Comment:
It would be nice to see a sort of validation here that ensures `end > start` 😎

Reply:
I have included validation in the model, 
end:{
        type: Date,
        required: true,
        validate: {
            validator: function(v){
            return v > this.start
            },
            message: "End Date must later than Start Date"
        }
    },

Reply:
Invalid:
![image](https://user-images.githubusercontent.com/37051668/135699047-fe1bf656-27ca-4ced-86d1-3713708e46eb.png)
Valid:
![image](https://user-images.githubusercontent.com/37051668/135699137-4e56a5e9-8369-4cbd-857b-9aad7d4a2ca8.png)